gnc/sub8_perception/src/sub8_pcl/cv_utils.cc

```diff
@@ -8,6 +8,11 @@ cv::Point contour_centroid(Contour& contour) {
   return center;
 }
 
+bool larger_contour(const Contour &c1, const Contour &c2){
+	if(cv::arcLength(c1, true) > cv::arcLength(c2, true)) return true;
```

Comment:
This might be inconsistent for a comparing a jagged segmentation to a smooth one (i.e. the jagged will have apparently more length, but about the same area)


Consider simplifying contours with approxpolyDP, or using area as a proxy for size